Examples/Physics_applications/beam-beam_collision/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _examples-beam-beam_collision:
+
+Beam-beam collision
+====================
+
+This example shows how to simulate the collision between two ultra-relativistic particle beams.
+This is representative of what happens at the interaction point of a linear collider.
+We consider a right-propagating electron bunch colliding against a left-propagating positron bunch.
+
+We turn on the Quantum Synchrotron QED module for photon emission (also known as beamstrahlung in the collider community) and
+the Breit-Wheeler QED module for the generation of electron-positron pairs (also known as coherent pair generation in the collider community).
+
+To solve for the electromagnetic field we use the nodal version of the electrostatic relativistic solver.
+This solver computes the average velocity of each species, and solves the corresponding relativistic Poisson equation (see the WarpX documentation for `warpx.do_electrostatic = relativistic` for more detail). This solver accurately reproduced the subtle cancellation that occur for some component of the ``E + v x B`` terms which are crucial in simulations of relativistic particles.
+
+
+This example is based on the following paper :cite:t:`ex-Yakimenko2019`.
+
+
+Run
+---
+
+The PICMI input file is not available for this example yet.
+
+For `MPI-parallel <https://www.mpi-forum.org>`__ runs, prefix these lines with ``mpiexec -n 4 ...`` or ``srun -n 4 ...``, depending on the system.
+
+.. literalinclude:: inputs
+   :language: ini
+   :caption: You can copy this file from ``Examples/Physics_applications/beam-beam_collision/inputs``.
+
+
+Visualize
+---------
+
+The figure below shows the number of photons emitted per beam particle (left) and the number of secondary pairs generated per beam particle (right).
```

Comment:
As a follow-up PR, could you post the Python script that was used to obtain this figure?

Reply:
I thought about this but decided not to do it because I'm using a digitized version of the data from Yakimenko's paper. 
Should I upload those files too?
Otherwise, I could add the portion of the python script that deals with the WarpX data only.